dbt_schema_builder/builder.py

```diff
@@ -185,27 +179,59 @@ def get_app_schema_configs(self):
         with open(schema_config_file_path, "r") as f:
             config = yaml.safe_load(f)
 
-        validate_schema_config(config)
+        self.validate_schema_config(config)
 
         return config
 
-    def get_project_dirs(self):
-        """
-        Find the dbt project directory based on the command line inputs.
-        """
-        source_project_path = os.getcwd()
-        destination_project_path = os.path.join(
-            source_project_path, self.args.destination_project
-        )
-
-        for project_path in [source_project_path, destination_project_path]:
-            if not os.path.exists(os.path.join(project_path, "dbt_project.yml")):
-                raise Exception(
-                    "fatal: {} is not a dbt project. Does not exist or is missing a "
-                    "dbt_project.yml file.".format(project_path)
-                )
-
-        return source_project_path, destination_project_path
+    @staticmethod
+    def validate_schema_config(config):
+        """
+        Read through an app-schema config dict, making sure that it meets certain
+        expectations before proceeding. If not, raise an exception to prevent
+        invalid schemas from being built
+        """
+        valid_keys = ['EXCLUDE', 'INCLUDE', 'SOFT_DELETE']
+        for _, app_config in config.items():
+            for schema, schema_config in app_config.items():
+                # This represents the case in which an application schema does
+                # not have any special logic concerning which tables to include
+                # or exclude
+                if not schema_config:
+                    continue
+                keys = schema_config.keys()
+                if 'EXCLUDE' in keys and 'INCLUDE' in keys:
+                    raise InvalidConfigurationException(
+                        "{} has both an EXCLUDE and INCUDE section".format(
+                            schema
+                        )
+                    )
+                if 'EXCLUDE' not in keys and 'INCLUDE' not in keys and 'SOFT_DELETE' not in keys:
+                    raise InvalidConfigurationException(
+                        "{} must have either an EXCLUDE or INCUDE section".format(
```

Comment:
if this conditional can fail due to the absence of `SOFT_DELETE` we should tell the user here as well